source/opt/instruction.cpp

```diff
@@ -171,20 +171,17 @@ uint32_t Instruction::NumInOperandWords() const {
 }
 
 bool Instruction::HasBranchWeights() const {
-  if (opcode_ == SpvOpBranchConditional &&
-      NumOperands() == kOpBranchConditionalWithWeightsNumOperands) {
-    return true;
-  }
-
-  return false;
```

Comment:
We might sometimes end up with code like this because it is easier to add a break point that breaks anytime this function returns true.  You break on "return true".  With your code, we would have to add a conditional breakpoint, which is much more tedious to add.

The change looks cleaner, so this is another example of something that we treat as a personal preference.

Reply:
Done.